### DISCUSSION TITLE
[Bug] Fetching GitLab Projects API results in

### DISCUSSION THREAD

**Discussion by undefinedhuman** [grafana/grafana-infinity-datasource#1039] [Specific APIs] [answered]
## **Describe the bug**

I am trying to setup a really simple HTTP data source to query the GitLab Projects API. 

## **Steps to reproduce**

Provide steps to reproduce the issue.

1. Configure a Infinity Data Source with a valid API key via the Bearer Authentication. (same error if you configure no auth)
2. Add `https://gitlab.com` as an allowed host
3. Click "Explore data"
4. Use Type "JSON", Parser "Default", Source "URL", Format "Table"
5. GET Request against "https://gitlab.com/api/v4/projects"

## **Screenshots**

![Data Source Config](https://github.com/user-attachments/assets/746e834a-769f-4373-b8cf-189c2ab57dae)

![Error Screen](https://github.com/user-attachments/assets/4f9130b5-3ede-42f9-bf4b-1a22175e4eb0)

## **Version Details:**

- Grafana version: - (Grafana Cloud)
- Plugin version: v2.11.0

**Comment by f0rkth1s** [comment]
I am also having this same exact issue.

**Reply by f0rkth1s** [reply]
Has there been any update on this particular issue?

**Comment by undefinedhuman** [comment, marked as answer]
Fixed in 2.11.3
